.github/ISSUE_TEMPLATE/release-version.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Release version
+about: Prepare and publish a release
+title: "[RELEASE] vX.Y.Z"
+labels: type/release
+assignees: ''
+
+---
+
+#### Release checklist
+
+* [ ] Default OS cloud images updated
+* [ ] Support branch created
+* [ ] CI pipelines for support branch created
+* All automated tests passed
```

Comment:
When it comes to all test passing, maybe we need to make some mention of the different OSses these tests need to pass for?

Reply:
I've clarified that this point applies to QA pipelines (contain tests for Ubuntu and RHEL).